api/src/main/java/jakarta/json/spi/JsonProvider.java

```diff
@@ -57,15 +79,27 @@ protected JsonProvider() {
     }
 
     /**
-     * Creates a JSON provider object. The provider is loaded using the
-     * {@link ServiceLoader#load(Class)} method. If there are no available
-     * service providers, this method returns the default service provider.
+     * Creates a JSON provider object.
+     *
+     * Implementation discovery consists of following steps:
+     *
+     * 1. If the system property {@link #JSONP_PROVIDER_FACTORY} exists,
+     *    then its value is assumed to be the provider factory class.
+     *    This phase of the look up enables per-JVM override of the JsonProvider implementation. 
+     * 2. The provider is loaded using the {@link ServiceLoader#load(Class)} method.
+     * 3. If there are no available service providers, this method returns the {@link #DEFAULT_PROVIDER}
+     *    service provider.
```

Comment:
this step is wrong - for `-api` artifact, default provider is not available, also the field is not public thus nobody can rely on/use that.

The wording should be like:
_If all the steps above fail, then the rest of the look up is unspecified. That said, the recommended behavior is to simply look for some hard-coded platform default Jakarta JSON Processing implementation. This phase of the look up is so that a platform can have its own Jakarta JSON Processing implementation as the last resort._

also note, that one can use html tags such as ol, li in the javadoc comment

Reply:
I think it is correct now. Thank for your patience :1st_place_medal: